src/app/frontend/common/components/label/label.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../../variables';
+
+.kd-label-chips {
+  > .md-chips {
+    box-shadow: none;
+    padding: inherit;
+  }
+
+  .md-chip {
+    border-radius: 0;
+    font-size: $caption-font-size-base;
+    height: $caption-font-size-base + 6;
+    line-height: $caption-font-size-base + 6;
```

Comment:
Sure. It can be refactored later when some appropriate name is proposed. Feel free to change it later in some PR. :)